README.md

```diff
@@ -26,6 +26,13 @@ You can also explore the dataset using the hosted [ThreatExchange UI](https://de
 
 To request access to ThreatExchange, please submit an application via [https://developers.facebook.com/products/threat-exchange/](https://developers.facebook.com/products/threat-exchange/).
 
+## Contributing
+
+We welcome contributions! See [CONTRIBUTING](https://github.com/facebook/ThreatExchange/blob/master/CONTRIBUTING.md) for details on how to get started, and our [Code of Conduct](https://github.com/facebook/ThreatExchange/blob/master/CODE_OF_CONDUCT.md).
+
 ## License
 
-Please see `./LICENSE`.
+Haxl uses the BSD 3-clause License, as found in the [LICENSE](https://github.com/facebook/ThreatExchange/blob/master/LICENSE) file.
```

Comment:
We do need to work a bit on the license language in the Readme, but right now there are multiple license files within this repo for different parts of the code base. As such, could you remove the change to the License section of the Readme and we will address that as a separate issue.